### DISCUSSION TITLE
About fabrics that include gesture modules.

### DISCUSSION THREAD

**Discussion by IS-hanabusa** [fabricjs/fabric.js#9054] [Q&A]
Can I turn off object rotation and resizing when we run a fabric that contains a gesture module?

I want to move objects around when  use  iPad. 
But I don't want rotation and scaling.

**Comment by IS-hanabusa** [comment]
I think I could have done it if I set lockRotation,lockScalingX,lockScalingY on the object. Thanks.

**Comment by ShaMan123** [comment]
You can also remove the relevant controls

**Reply by IS-hanabusa** [reply]
I will remove that and try it. Thanks!
